proposals/devcontainer-templates-distribution.md

```diff
@@ -0,0 +1,120 @@
+
+Source code for the set follows the example file structure below:
+
+```
+.
+├── README.md
+├── src
+│   ├── dotnet
+│   │   ├── devcontainer-template.json
+│   │   ├── .devcontainer.json
+│   │   ├── ...
+|   ├
+│   ├── docker-from-docker
+│   │   ├── devcontainer-template.json
+│   │   ├── .devcontainer
+│   │       ├── devcontainer.json
+│   │       ├── Dockerfile
+│   │       └── ...
+│   │   ├── ...
+|   ├
+│   ├── go-postgres
+│   │   ├── devcontainer-template.json
+│   │   ├── .devcontainer
+│   │       ├── devcontainer.json
+│   │       ├── docker-compose.yml
+│   │       ├── Dockerfile
+│   │       └── ...
+│   │   ├── ...
+```
+
+...where `src` is a directory containing a sub-folder with the name of the Template (e.g. `src/dotnet` or `src/docker-from-docker`) with at least a file named `devcontainer-template.json` that contains the Template metadata, and a `.devcontainer.json` (or `.devcontainer/devcontainer.json`) that the supporting tools will drop into an existing project or folder.
+
+Each sub-directory should be named such that it matches the `id` field of the `devcontainer-template.json`.  Other files can also be included in the Templates's sub-directory, and will be included during the [packaging step](#packaging) alongside the two required files.  Any files that are not part of the Templates's sub-directory (e.g. outside of `src/dotnet`) will not included in the [packaging step](#packaging).
+
+## Versioning
+
+Each Template is published with only the `latest` tag. Tooling that handles releasing Templates will republish the `latest` tag every time a new release is created.
+
+## Packaging
+
+Templates are distributed as tarballs. The tarball contains the entire contents of the Template sub-directory, including the `devcontainer-template.json`, `.devcontainer.json` (or `.devcontainer/devcontainer.json`), and any other files in the directory.
+
+The tarball is named `devcontainer-template-<id>.tgz`, where `<id>` is the Templates's `id` field.
+
+A reference implementation for packaging and distributing Templates is provided as a GitHub Action (https://github.com/devcontainers/action).
+
+### devcontainer-collection.json
+
+The `devcontainer-collection.json` is an auto-generated metadata file.
+
+| Property | Type | Description |
+| :--- | :--- | :--- |
+| `sourceInformation` | object | Metadata from the implementing packaging tool. |
+| `templates` | array | The list of Templates that are contained in this collection.|
+
+Each templates's `devcontainer-template.json` metadata file is appended into the `templates` top-level array.
+
+## Distribution
+
+There are several supported ways to distribute templates.  Distribution is handled by the implementing packaging tool.
+
+A user can add the Templates in to their projects as defined by the [Templates Supporting Tools](./devcontainer-templates.md/#adding-a-template-to-a-project-or-codespace).
```

Comment:
```suggestion
A user can add Templates in to their projects as defined by the [Templates Supporting Tools](./devcontainer-templates.md/#adding-a-template-to-a-project-or-codespace).
```

Unless we're referring to a specific set of Templates (which would thus be "The" Templates?)